pages/room/index.js

```diff
@@ -31,6 +58,9 @@ const CreateRoom = ({ spotifyAuthURL, spotifyCode, newSession }) => {
   const [client, setClient] = useRecoilState(clientState.client);
   const setSpotifyRefresh = useSetRecoilState(spotifyState.refresh);
 
+  const [createSpace, { data, loading, error }] = useMutation(CREATE_SPACE);
+  // const [createMessage, { data, loading, error }] = useMutation(CREATE_MESSAGE);
```

Comment:
The createMessage mutation works when you call createMessage() on line 97, but the createSpace hook does not, even though the createSpace resolver works, having tested it with the GUI